pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -215489,7 +215489,6 @@ self: {
        license = lib.licenses.gpl2Only;
        hydraPlatforms = lib.platforms.none;
```

Comment:
```suggestion
```

When marking a haskell-package unbroken, you have to make sure to remove this `hydraPlatforms` line as well, otherwise Hydra won't build the package.

This would happen automatically the next time we regenerated the package set, but I imagine you'd want this to happen immediately for this PR as well.

Reply:
Cool, thanks for letting me know.